assets/sass/06-components/pagination.scss

```diff
@@ -24,6 +24,13 @@
 	}
 
 	.nav-links {
+
+		.nav-next a,
+		.nav-previous a {
+			display: flex;
+			flex-direction: column;
+		}
```

Comment:
I had to switch to using flex box here to override some annoying alignment issues with the next post item: 

Before|After
---|---
<img width="318" alt="Screen Shot 2020-10-15 at 10 17 26 AM" src="https://user-images.githubusercontent.com/1202812/96142371-041e1980-0ed0-11eb-9a7a-c375783b70f7.png">|<img width="315" alt="Screen Shot 2020-10-15 at 10 17 48 AM" src="https://user-images.githubusercontent.com/1202812/96142385-07b1a080-0ed0-11eb-95e5-40762a2cef7c.png">